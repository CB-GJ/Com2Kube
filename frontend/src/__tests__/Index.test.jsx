import React from "react"
import { shallow } from "enzyme"
import Index from "../pages/index"

it("renders correctly enzyme", () => {
  const wrapper = shallow(<Index />)

  expect(toJson(wrapper)).toMatchSnapshot()
})
