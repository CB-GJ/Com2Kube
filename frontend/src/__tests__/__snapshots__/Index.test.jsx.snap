// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly enzyme 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="MuiPaper-root MuiAlert-root MuiAlert-standardInfo MuiPaper-elevation0"
    role="alert"
  >
    <div
      className="MuiAlert-icon"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M11,9H13V7H11M12,20C7.59,20 4,16.41 4,12C4,7.59 7.59,4 12,4C16.41,4 20,7.59 20, 12C20,16.41 16.41,20 12,20M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10, 10 0 0,0 12,2M11,17H13V11H11V17Z"
        />
      </svg>
    </div>
    <div
      className="MuiAlert-message"
    >
      <div
        className="MuiTypography-root MuiAlertTitle-root MuiTypography-body1 MuiTypography-gutterBottom"
      >
        index.betaTitle
      </div>
      index.betaMessage
      <div
        className={
          Object {
            "margin": "1.2rem",
          }
        }
      >
        <button
          alt="download docker-compose"
          className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="submit"
        >
          <span
            className="MuiButton-label"
          >
            <span
              className="MuiButton-startIcon MuiButton-iconSizeMedium"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.59 9H15V4c0-.55-.45-1-1-1h-4c-.55 0-1 .45-1 1v5H7.41c-.89 0-1.34 1.08-.71 1.71l4.59 4.59c.39.39 1.02.39 1.41 0l4.59-4.59c.63-.63.19-1.71-.7-1.71zM5 19c0 .55.45 1 1 1h12c.55 0 1-.45 1-1s-.45-1-1-1H6c-.55 0-1 .45-1 1z"
                />
              </svg>
            </span>
            fileSample.downloadBtn
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="MuiContainer-root makeStyles-text-2 MuiContainer-maxWidthLg"
  >
    <p>
      index.introduction
    </p>
    <p>
      index.steps
    </p>
    <ol
      className="makeStyles-li-3"
    >
      <li>
        index.step1
      </li>
      <li>
        index.step2
      </li>
      <li>
        index.step3
      </li>
    </ol>
    <span>
      index.disclaimer
    </span>
  </div>
  <div
    className="MuiBox-root MuiBox-root-129"
  >
    <div>
      <form
        className={
          Object {
            "display": "flex",
            "flexDirection": "column",
            "width": "100%",
          }
        }
        encType="multipart/form-data"
        onSubmit={[Function]}
      >
        <div
          className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiInputBase-root MuiInput-root MuiInput-underline"
            onClick={[Function]}
            required={true}
          >
            <input
              className="MuiInputBase-input MuiInput-input"
              id="compose_file"
              name="compose_file"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={true}
              type="file"
            />
          </div>
          <div
            className="MuiBox-root MuiBox-root-267"
          >
            <button
              alt="convert button"
              className="MuiButtonBase-root MuiButton-root MuiButton-outlined"
              disabled={false}
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="submit"
              value="upload"
            >
              <span
                className="MuiButton-label"
              >
                fileUpload.submitBtn
              </span>
            </button>
          </div>
        </div>
        <div
          className="MuiContainer-root MuiContainer-maxWidthLg"
        >
          <p />
        </div>
      </form>
    </div>
  </div>
</div>
`;
